script/configs/allowed_unpinned_deps.yaml

```diff
@@ -41,6 +41,7 @@
 - io
 - js
 - json_serializable
+- leak_tracker
```

Comment:
Added to use [forceGC](https://pub.dev/documentation/leak_tracker/latest/leak_tracker/forceGC.html)